### PR TITLE
Comment NL28

### PR DESCRIPTION
**Pull request by Aurige**
Presentation added to DesinationDisplay (and Variant) + Branding

### CODE REVIEW

File: xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd [hunk 1]
```diff
@@ -756,6 +761,11 @@ Rail transport, Roads and Road transport
 				</xsd:annotation>
 			</xsd:element>
 			<xsd:group ref="DestinationDisplayNameGroup"/>
+			<xsd:element name="Presentation" type="PresentationStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Presentation values to use when rendering DESTINATION DISPLAY VARIANT, such as a colour.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd:768]: This part of the pull request has not yet seen in master. @nick-knowles

**Comment by skinkie** [comment]
As pointed out above, one single change regarding presentation has not been added to master yet. But more troublesome, I am not able to find any definition of Branding in Part 1.

**Review by Aurige — COMMENTED** [review]
_Branding_ was initially just a _TypeOfValue_, so was not requesting much documentation... but with the addition of _Presentation_ in v1.1 it now definitely does :-( well spotted

File: xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd (hunk 1 above)
Comment (skinkie) [reply xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd:768]: I'll suggest here to approach it as new pull request. That prevents a mess.

File: xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd [hunk 2]
```diff
@@ -602,6 +602,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Alternative name and Code to use in a specifc context. +V1.1</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="Presentation" type="PresentationStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Presentation values to use when rendering DESTINATION DISPLAY, such as a colour.</xsd:documentation>
+				</xsd:annotation>
```
Comment (ue71603) [inline xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd:608]: This part is with a slight variation in the documentation already in master.

File: xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd:768]: This part is not in master yet. @Aurige : how to proceed.

**Review by ue71603 — APPROVED** [review]

File: xsd/netex_framework/netex_responsibility/netex_dataSource_version.xsd [hunk 3]
```diff
@@ -177,6 +177,7 @@ References to a DATA SOURCE are useful in an interoperated computer system.</xsd
 						</xsd:sequence>
 						<xsd:sequence>
 							<xsd:group ref="TypeOfValueGroup"/>
+							<xsd:element name="Presentation" type="PresentationStructure" minOccurs="0"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_responsibility/netex_dataSource_version.xsd:180]:
```suggestion
```

**Review by skinkie — APPROVED** [review]

**Comment by Aurige** [comment]
Documentation updated, including extra alignment with XSD

**Pull request merged**